.circleci/config.yml

```diff
@@ -14,7 +14,7 @@ jobs:
           version: 17.07.0-ce
 
       - run: go get -u github.com/golang/dep/cmd/dep
-      - run: dep ensure
+      - run: dep ensure -update
```

Comment:
This will update the Gopkg.lock inside CI which is not a good idea. The structure is a bit confusing right now because there is the outer Gopkg.lock (which is what this is using) and the inner chart-repo cmd's Gopkg.lock. The reason the tests might have been failing is because the outer Gopkg.lock hasn't been updated, so you need to run `dep ensure -update` locally in the root and then commit the Gopkg.lock. Does that make sense?

Reply:
The reason the tests have been failing is because the latest update from `kubeapps/common` package that you've made yesterday hasn't been fetched into `kubeapps/chartsvc` when running CI. The good thing I am seeing at `dep` is the flexibility capability to specify dep versions in each individual folder, however that is also a disadvantage at some point when it messes up the whole code structure.

Your suggestion makes sense. I will update the outer Gopkg.lock and remove `-update`. However, I suggest we keep restructuring the source code and keep the vendoring at only the root-level directory (like other dependency management tools)

Reply:
Yeah, I agree, the only issue is that we'll need to figure out how to build the Docker image for the chart-repo cmd. I guess we could just build the binary in the CI and then just have the Dockerfile COPY it.